docs/command_line_tool/phylum_extension_new.md

```diff
@@ -7,13 +7,20 @@ hidden: false
 Create a new extension
 
 ```sh
-phylum extension new <PATH>
+Usage: phylum extension new [OPTIONS] <PATH>
 ```
 
-### Details
+### Arguments
 
-This command will create a "Hello, World!" extension skeleton that can be
-modified for a specific use. See the [Extension Quickstart] guide for more
-detail.
```

Comment:
These details were dropped. Was that on purpose? It might be worth adding them in a template file.